### PR TITLE
Fix error next command in autotune

### PR DESCRIPTION
**Pull request by MagoKimbra**
Now send only temperature for T or B, but not send OK.
Host interprets the line to show the right temperature, but not in
response to M105 then stop and send commands until it ends the autotune.

### CODE REVIEW

**Comment by thinkyhead** [comment]
I guess we just need to pass this by a few host geeks… @repetier @foosel @daid @kliment and…?

I have added a new label "Hosts & Protocols" to alert host and slicer maintainers that an issue pertains to their areas of expertise.

**Comment by foosel** [comment]
Am I correct in assuming that the current output produced by Marlin for `M303` would be something along the lines of this?

```
ok T:<input>@<p>
ok T:<input>@<p>
ok T:<input>@<p>
ok T:<input>@<p>
Resend:<N+1>
ok
```

If so, then this patch in fact fixes a bug, since `ok` should only be sent when the command is actually done - one `ok` per command, and at the end. The fact that the current implementation also does some resend requesting hints at the implementation having an issue.

tldr: Unless I'm misunderstanding the intention of the patch and the output of `M303` in the current implementation, this is fine and actually necessary.

PS: Thanks for the label. Do you know of any way to subscribe only to those? Not seeing a way to create an RSS feed from search or something... Maybe I'll have to whip up some quick script.

**Comment by daid** [comment]
Cura never uses M303 on its own. But it's good to be kept in the loop.

**Comment by thinkyhead** [comment]
@foosel I can't find any way to actually _subscribe_ to an issue label, but you can bookmark [this link](https://github.com/MarlinFirmware/Marlin/labels/Hosts%20%26%20Protocols) and check it periodically. The Github web API also supports querying issues by label, if you feel like whipping up a fetch script.

**Comment by thinkyhead** [comment]
@MagoKimbra Good catch. This is one subtle bug!

**Comment by MagoKimbra** [comment]
Thanks @thinkyhead
Il 12/Giu/2015 13:49, Scott Lahteine notifications@github.com ha scritto:

**Comment by foosel** [comment]
@thinkyhead yeah, I know about the API, I just hoped that you might have stumbled across some existing solution already that I haven't ;) Looks like some script then, when I find the time and motivation for that
